RecoEgamma/PhotonIdentification/python/PhotonIDValueMapProducer_cff.py

```diff
@@ -0,0 +1,9 @@
+from RecoEgamma.PhotonIdentification.photonIDValueMapProducer_cfi import *
```

Comment:
please use the same capitalization in the cff files as in the cfi. Lower case is preferred.